net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild

```diff
@@ -17,13 +17,16 @@ IUSE="nls xinetd"
 RDEPEND="sys-libs/ncurses:="
 DEPEND="${RDEPEND}
 	!net-misc/netkit-telnetd
-	xinetd? ( sys-apps/xinetd )
-	virtual/pkgconfig"
+	xinetd? ( sys-apps/xinetd )"
```

Comment:
Also, surely this is needed for at least runtime?

Reply:
:thinking: you're right, I think this should be RDEPEND only